packages/vite-plugin-svelte/src/utils/vite-plugin-svelte-stats.ts

```diff
@@ -85,6 +86,31 @@ function formatPackageStats(pkgStats: PackageStats[]) {
 	return table;
 }
 
+async function parseClosestNamedPackage(
+	file: string,
+	dir?: string
+): Promise<{ name: string; path: string }> {
+	const pkgPath = await findClosestPkgJsonPath(dir ?? file);
+	if (pkgPath) {
+		const name = JSON.parse(readFileSync(pkgPath, 'utf-8')).name;
+		if (!name) {
+			return parseClosestNamedPackage(file, dirname(pkgPath));
+		}
+		const path = pkgPath.replace(/package.json$/, '');
+		return { name, path };
+	} else {
+		// no package.json found with a name in it, give up.
+		const path = dirname(file) + '/';
```

Comment:
maybe we should use `path.join` to be safe. needs to update the import though.
```suggestion
		const path = path.join(dirname(file), '/');
```

Reply:
it was worse, vitefu doesn't normalize so cached paths were not found by the `file.startsWith(path)` check. Updated to include normalization